File: app/src/main/java/ro/alexmamo/weatherapp/cities/Cities.java
package ro.alexmamo.weatherapp.cities;

import java.util.ArrayList;
import java.util.List;

import ro.alexmamo.weatherapp.cities.models.City;

class Cities {
    static List<City> getCityList() {
        List<City> cityList = new ArrayList<>();
        cityList.add(new City("Brasov", "https://www.alexmamo.ro/apps/WeatherApp/CityImages/Brasov.jpg"));
        cityList.add(new City("Bucharest", "https://www.alexmamo.ro/apps/WeatherApp/CityImages/Bucharest.jpg"));
        cityList.add(new City("Constanta", "https://www.alexmamo.ro/apps/WeatherApp/CityImages/Constanta.jpg"));
        return cityList;
    }
}

File: app/src/main/java/ro/alexmamo/weatherapp/cities/Cities.kt
package ro.alexmamo.weatherapp.cities

import java.util.ArrayList

import ro.alexmamo.weatherapp.cities.models.City

internal object Cities {
    val cityList: List<City> get() {
        val cityList = ArrayList<City>()
        cityList.add(City("Brasov", "https://www.alexmamo.ro/apps/WeatherApp/CityImages/Brasov.jpg"))
        cityList.add(City("Bucharest", "https://www.alexmamo.ro/apps/WeatherApp/CityImages/Bucharest.jpg"))
        cityList.add(City("Constanta", "https://www.alexmamo.ro/apps/WeatherApp/CityImages/Constanta.jpg"))
        return cityList
    }
}